src/BlockSync.ts

```diff
@@ -193,52 +216,70 @@ export class BlockSync {
     const client = this.getRpcClient(header);
     const api = await client.at(header.hash);
     const events = await api.query.system.events();
-    const { rotation: _r, ...cohortIds } = await this.accountMiners.onBlock(
+    const { rotation: _r, ...cohortActivationAtFrameIds } = await this.accountMiners.onBlock(
```

Comment:
Why is this called cohortActivationAtFrameIds? Isn't this just returning the earnings for a frameId that are grouped by cohortId?